### ISSUE TITLE
Add support for "newTabPosition" API

### ISSUE THREAD

**Issue by Keith94** [autonome/Always-Right#22]
https://bugzilla.mozilla.org/show_bug.cgi?id=1344749 was fixed today; I'm wondering if this add-on should use the new API to control new tabs position.

>This patch implements the preference "browser.tabs.insertAfterCurrent" which,
when set to true, will cause all tabs (related and unrelated) to be opened next
to the current tab.

>It also implements the browserSettings API "newTabPosition", which allows
extensions to control both "browser.tabs.insertRelatedAfterCurrent", and
"browser.tabs.insertAfterCurrent" via values for "afterCurrent",
"relatedAfterCurrent" and "atEnd".

**Comment by peteruithoven** [comment]
Looks like you did in the following commit? 
https://github.com/autonome/Always-Right/commit/83ad05b14a3022af4182446c9cdb29336ac6d985

And this it seems to work great in Firefox nightly 62, tabs are opened directly after the current instead of first being opened at the end and then moved.

**Comment by Keith94** [comment]
@mems Yes. For more info see [ghacks](https://www.ghacks.net/2018/05/10/open-tabs-in-firefox-to-the-right-of-the-current-tab/).
